modules/sql/server/vulnerability-assessment/.bicep/nested_storageRoleAssignment.bicep

```diff
@@ -0,0 +1,17 @@
+param storageAccountName string
```

Comment:
Please rename folfer to "modules" as we just changed it as part of the Rbac PRs

Reply:
don't mind doing that. We should then align the SQL MI one too. Agree? I can do it using this PR as well

Reply:
@AlexanderSehr updated both modules. SQL Serv ran successfully. SQL MI static tests passed (module dep in progress)